modules/oncodrivefml/1.0/etc/oncodrivefml.conf

```diff
@@ -0,0 +1,33 @@
+[genome]
+build = 'hg19'
```

Comment:
does this need to be adjusted if running in hg38 mode?

Reply:
Yes - I added more detailed information to the default config